ioctl/newcmd/alias/aliasremove_test.go

```diff
@@ -0,0 +1,30 @@
+package alias
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/util"
+	"github.com/iotexproject/iotex-core/test/mock/mock_ioctlclient"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewAliasRemoveCmd(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+	client := mock_ioctlclient.NewMockClient(ctrl)
+	client.EXPECT().SelectTranslation(gomock.Any()).Return("%s is removed", config.English).Times(6)
+	cfg := config.Config{
+		Aliases: map[string]string{
+			"a": "io1uwnr55vqmhf3xeg5phgurlyl702af6eju542sx",
+			"b": "io1uwnr55vqmhf3xeg5phgurlyl702af6eju542sx",
+			"c": "io1uwnr55vqmhf3xeg5phgurlyl702af6eju542s1",
+		},
+	}
+	client.EXPECT().Config().Return(cfg).AnyTimes()
+	cmd := NewAliasRemove(client)
+	res, err := util.ExecuteCmd(cmd, "a")
```

Comment:
incomplete unit test: how do you verify that the corresponding alias has been removed?

Reply:
@CoderZhi 
supplementary test